addons/payment/models/payment_acquirer.py

```diff
@@ -684,6 +684,10 @@ def form_feedback(self, data, acquirer_name):
 
         return True
 
+    @api.model
+    def form_confirmation(self, invoice_id=False):
```

Comment:
Well the API is strange. I would give this method kwargs and let people deal with it.